firefox-ios/Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -45,6 +45,12 @@ class FakespotViewController:
     var themeManager: ThemeManager
     var themeObserver: NSObjectProtocol?
     private var viewModel: FakespotViewModel
+    private lazy var telemetryAdsExposure: ((Bool) -> Bool) = { [weak self] areAdsEmpty in
+        if areAdsEmpty {
+            return (self?.fakespotState?.notExposedToAdsEvent ?? false)
```

Comment:
Just wanted to express my concern, I think the use of optional state creates a bit of a mess having to default variables. With booleans it's either true or false but with Strings and other types it can go to invalid states.